components/server/src/ome/services/RenderingBean.java

```diff
@@ -774,6 +776,90 @@ public long saveAsNewSettings() {
 
     /**
      * Implemented as specified by the {@link RenderingEngine} interface.
+     *
+     * @see RenderingEngine#updateSettings(RenderingDef)
+     */
+    @RolesAllowed("user")
+    public void updateSettings(RenderingDef settings) {
+        if (settings == null) {
+            return;
+        }
+
+        // Emulate setModel(RenderingModel)
+        RenderingModel model = settings.getModel();
+        if (model != null) {
+            setModel(model);
```

Comment:
It's probably not a semantic that we have ever explicitly defined and I don't know what would happen but technically `null` can be passed through to `RenderingBean#setModel(RenderingModel)` now. We should be sure that this is still adhering to those semantics and that people aren't calling `setModel(null)` and expecting a specific behaviour.